docker/dev/Dockerfile

```diff
@@ -7,6 +7,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 
 # CodeCompass development dependencies.
 RUN set -x && apt-get update -qq \
+  && apt-get install -y postgresql-14 \
   && apt-get -y install --no-install-recommends \
```

Comment:
Please extend the existing `apt-get install` statement instead. You can add this new package to the end of the list.